MFVNeutralino/test/TrackMover/WeightFiles1D.C

```diff
@@ -0,0 +1,69 @@
+#include <vector>
+#include <string>
+#include <iostream>
+
+void MakeWeightPlots(bool Is_bkg, const char* boson, int mg, int ctau, int year)
+{
+  TString fns;
+  //This is for the previous signal samples
+  //This is for the new signal samples
+  fns.Form("~/nobackup/crabdirs/TrackMoverMCTruth_StudyMinijets_NoPreSelRelaxBSPVetoMissLLPVetoTrkJetByMiniJetHistsOnnormdzUlv30lepmumv6/%sHToSSTodddd_tau%imm_M%02i_%i.root",boson,ctau/1000,mg,year);
```

Comment:
Whose home dir is this based on? Might be good to have the full path, in case someone else were to ever run this (and true for other cases in this file and WeightFiles1Dkin.C)

Reply:
And the 2D files too. It's a general comment, so anywhere else that it applies too!